dist/routable-component.js

```diff
@@ -100,20 +100,20 @@ var routableComponent =
 	   * constructor
 	   * @param {Object<EventEmitter>} [components] コンポーネントの連想配列
 	   * @param {RoutableComponentRoutes} routes ルーティング
-	   * @param {Object<RoutableComponentController>} controller_classes コントローラクラスの連想配列
+	   * @param {Object<RoutableComponentController>} controllerClasses コントローラクラスの連想配列
 	   */
 	
-	  function RoutableComponent(components, routes, controller_classes) {
+	  function RoutableComponent(components, routes, controllerClasses) {
 	    (0, _classCallCheck3.default)(this, RoutableComponent);
 	
 	    var _this = (0, _possibleConstructorReturn3.default)(this, (0, _getPrototypeOf2.default)(RoutableComponent).call(this));
 	
 	    _this._routes = routes;
-	    _this._controller_classes = controller_classes;
+	    _this._controllerClasses = controllerClasses;
```

Comment:
__[eslint]__
* Error - Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
* Error - Expected indentation of 2 space characters but found 0. (indent)

<!-- Comment by [SideCI](https://sideci.com) -->